packages/html-reporter/src/metadataView.tsx

```diff
@@ -18,21 +18,62 @@ import * as React from 'react';
 import './colors.css';
 import './common.css';
 import * as icons from './icons';
-import type { Metadata } from './index';
 import { AutoChip } from './chip';
 import './reportView.css';
 import './theme.css';
 
-export const MetadataView: React.FC<Metadata> = metadata => {
+interface Info {
+  'revision.id'?: string;
+  'revision.author'?: string;
+  'revision.email'?: string;
+  'revision.subject'?: string;
+  'revision.timestamp'?: number | Date;
+  'revision.link'?: string;
+  'ci.link'?: string;
+}
+
+export type GitCommitInfo =  (Info & { timestamp?: number }) | undefined;
```

Comment:
```suggestion
export type CommitInfo =  (Info & { timestamp?: number }) | undefined;
```

Reply:
You should merge it into the above Info, rename to Metainfo (Info is too generic)?